configuration/spring-config-server/src/main/resources/application.yml

```
@@ -6,13 +6,26 @@ spring:
+  rabbitmq:
+    host: localhost
+    port: 5672
+    username: guest
+    password: guest
 
+  # Here define actuator
+  management:
+    endpoints:
+      web:
+        exposure:
+          include : "*"
```

YAMLlint (1.37.1)

[warning] too many spaces before colon

(colons)